ocaml/idl/datamodel_host.ml

```diff
@@ -855,9 +855,10 @@ let host_query_ha = call ~flags:[`Session]
       ~name:"get_server_localtime"
       ~in_oss_since:None
       ~in_product_since:rel_cowley
+      ~lifecycle:[Published, rel_cowley, ""; Changed, rel_next, "Return string instead of datetime"]
       ~params:[Ref _host, "host", "The host whose clock should be queried"]
       ~doc:"This call queries the host's clock for the current time in the host's local timezone"
-      ~result:(DateTime, "The current local time")
+      ~result:(String, "The current local time")
```

Comment:
This will change the type of the XML-RPC response from `<dateTime.iso8601>` to `<string>`, which is a backwards-incompatible change that existing clients may (still) not tolerate.